src/backend/app/Console/Commands/DemoDataCreator.php

```diff
@@ -265,6 +267,22 @@ private function generateTransaction(): void {
             $token->save();
             $transactionData->token = $token;
 
+            // generate meter_token
+            $meterTokenData = [
+                'meter_id' => $randomMeter->id,
+                'token' => Str::random(30),
```

Comment:
Another nit here: Instead of random 30-char strings, can we can 12 digit numbers? `[0-9]` ? (Probably also applies to the code piece above)

Reply:
Should this be a random 12 digits number or we want to have a special token generator?

Reply:
I think for now a simple 12 random digits is fine 👍